nx.json

```diff
@@ -40,34 +40,16 @@
       ]
     }
   },
+  "affected": {
+    "defaultBase": "mainline"
+  },
   "targetDependencies": {
-    "build": [
-      {
-        "target": "build",
-        "projects": "dependencies"
-      }
-    ],
-    "release:mainline": [
-      {
-        "target": "release:mainline",
-        "projects": "dependencies"
-      }
-    ],
-    "upgrade": [
-      {
-        "target": "upgrade",
-        "projects": "dependencies"
-      }
-    ],
     "install": [
       {
         "target": "install",
         "projects": "dependencies"
       }
     ]
   },
```

Comment:
Should this be moved to `targetDefaults` as well?

Reply:
Ahh yep - good catch